lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -83,6 +86,35 @@ def test_bad_arg():
             > -1
         )
 
+    @staticmethod
+    @responses.activate
+    def test_meta_args():
+        """Test operation when irregular arguments and options are specified"""
+
+        TestResultsPush.add_http_mock_response()
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(
+            main,
+            args=[
+                TestResultsPush.TOKN_SWITCH,
+                TestResultsPush.TOKN_TEXT,
+                TestResultsPush.ACCESS_SWITCH,
+                TestResultsPush.ACCESS_TEXT,
+                TestResultsPush.META_SWITCH,
+                TestResultsPush.META_TEXT_TEST,
+                TestResultsPush.META_TEXT_FOO,
```

Comment:
What happens if we use this combination?
```
                TestResultsPush.META_SWITCH,
                TestResultsPush.META_TEXT_TEST + ',' + TestResultsPush.META_TEXT_FOO,
```
(i.e., `-m pbench.sat:FOO,pbench.test:TEST`).  Does that work as expected?

(I think that would be a better test than this one, which effectively provides two files rather than an orphaned metadatum.)

Reply:
It is not throwing any errors but I would love to experiment that in real environment.